src/lib/plugin.js

```diff
@@ -208,7 +209,11 @@ class FiveBellsLedger extends EventEmitter2 {
     }
 
     const reconnect = reconnectCore(() => {
-      return new WebSocket(wsUri)
+      return new WebSocket(wsUri, null, {
+        headers: {
+          authorization: 'Bearer ' + options.authToken
```

Comment:
This does not work out of the box in a browser. Let's stick with appending the token to the URL for now.